style.css

```diff
@@ -363,6 +363,36 @@ textarea.input {
   display: none;
 }
 
+.mobile-menu {
+  display: none;
+  z-index: 999;
+  position: absolute;
```

Comment:
- [ ] You have done an excellent job with your mobile menu :100: but when scrolling down, the mobile menu stays in the same place and I am able to see the rest of the content and it is supposed to keep on top of the content. I think you could use `position: fixed` in order to make to mobile menu stays on top, even when scrolling :pushpin: :smiley:
![image](https://user-images.githubusercontent.com/81522435/145885794-9c4688d4-3aec-4d11-ac49-6eaf3254cc43.png)